### PR TITLE
Replace travis with github actions

### PR DESCRIPTION
**Pull request by vmihailenco**

### CODE REVIEW

File: version.go [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package taskq
+
+// Version is the current release version.
+func Version() string {
+	return "0.0.0"
```
Comment (lilien1010) [inline version.go:5]: should we specifc the version as the tag version , eg `3.2.4`

File: .github/workflows/golangci-lint.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+name: golangci-lint
```
Comment (lilien1010) [inline .github/workflows/golangci-lint.yml:1]: maybe we can add a `.golangci.yml`?
```
linters-settings:
  funlen:
    lines: 100
    statements: 75
  gocritic:
    disabled-checks:
      - exitAfterDefer

linters:
  disable:
    - errcheck
  enable:
    - bodyclose
    - deadcode
    - depguard
    - dogsled
    - dupl
    - exhaustive
    - funlen
    - gochecknoinits
    - goconst
    - gocritic
    - gocyclo
    - gofmt
    - goimports
    - golint
    - goprintffuncname
    - gosimple
    - govet
    - ineffassign
    - misspell
    - nakedret
    - noctx
    - nolintlint
    - rowserrcheck
    - scopelint
    - staticcheck
    - structcheck
    - stylecheck
    - typecheck
    - unconvert
    - unparam
    - unused
    - varcheck
    - whitespace
```

File: version.go (hunk 1 above)
Comment (vmihailenco) [reply version.go:5]: This will be filled automatically by release.sh script

File: .github/workflows/golangci-lint.yml (hunk 2 above)
Comment (vmihailenco) [reply .github/workflows/golangci-lint.yml:1]: Not sure... It works without a config well enough and I don't want to learn & maintain golangci-lint config :) If you already have a config that you use in other projects - please send a PR.

**Pull request merged**